docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,358 @@
+
+Learn more in our [Charts Composition Documentation](/x/react-charts/composition/).
+
+## Enhanced License Management and Telemetry
+
+Managing your license has never been easier. Our redesigned [account page](https://mui.com/store/account/) on the store allows you to view your license keys, update number of seats, renew, upgrade and more, with minimal effort.
+
+<figure>
+  <img width="606" height="300" alt="Radar charts examples" src="/static/blog/mui-x-v8/accounts-page.png" loading="lazy">
+  <figcaption>Accounts Page</figcaption>
+</figure>
+
+In addition, we’ve integrated telemetry into our system to continuously enhance your experience with our products. Using this same infrastructure, you'll soon be able to manage your license without changing your key—thanks to server-side license validation.
+
+Telemetry is released as an opt-in feature. We recommend enabling it by adding the following code to your project, close to your license validation:
+
+```js
+import { muiXTelemetrySettings } from '@mui/x-license';
+
+muiXTelemetrySettings.enableTelemetry();
+```
+
+You can find more details in our [telemetry guide](/x/guides/telemetry/).
+
+## Priority Support – A New Support Service
+
+A few months ago, we launched Priority Support with a few customers, and now we’re excited to invite more users to take advantage of this service.
+
+Priority Support offers fast, dedicated assistance with guaranteed response times, ensuring that technical issues on both MUI X and MUI Core are addressed swiftly and efficiently.
+
+For full details on the benefits and service levels of Priority Support, please see our [Support SLA](https://mui.com/legal/technical-support-sla/).
+
+If you’re interested in upgrading to Priority Support, don’t hesitate to [contact our sales team](mailto:sales@mui.com) for more information or purchase it directly through your account page.
+
+## Support for Material UI
+
+When planning this major release, our goal was to support the last two major versions of Material UI. However, we faced a tough choice: continue supporting older versions (v5 and v6) or fully embrace the innovations of **Material UI v7**. With native ESM support, improved package management, and seamless integration with modern bundlers like Vite and webpack, v7 represents the future of Material UI. Supporting both wasn’t feasible, so we chose to move forward.
+
+As a result, **MUI X v8 now supports Material UI v7 natively**, ensuring your applications benefit from the latest improvements. We understand that change can be challenging, but migrating to v7 has been designed to be as smooth as possible—with comprehensive migration guides and GitHub support to assist you every step of the way.
+
```

Comment:
```suggestion

For users who want to upgrade to MUI X v8 while staying on Material UI v6, we've prepared a [guide](/x/migration/usage-with-material-ui-v5-v6/) to help you configure your project to keep using v6.
```

It is referring to [this guide](https://deploy-preview-17164--material-ui-x.netlify.app/x/migration/usage-with-material-ui-v5-v6/) from https://github.com/mui/mui-x/pull/17164